src/libmongoc/tests/unified/operation.c

```diff
@@ -242,19 +265,34 @@ operation_list_collections (test_t *test,
    bool ret = false;
    mongoc_database_t *db = NULL;
    mongoc_cursor_t *cursor = NULL;
+   bson_parser_t *parser = NULL;
+   bson_t *opts = NULL;
+
+   parser = bson_parser_new ();
+   bson_parser_allow_extra (parser, true);
+
+   opts = bson_copy (bson_parser_get_extra (parser));
```

Comment:
Similarly, since the parser is not parsing any input, this could be simplified to `opts = bson_new()`.

Reply:
Simplified to `opts = bson_new()` and removed `allow_extra` call.